kedro/logging.py

```diff
@@ -54,3 +55,15 @@ def __init__(self, *args: Any, **kwargs: Any):
             # fixed on their side at some point, but until then we disable it.
             # See https://github.com/Textualize/rich/issues/2455
             rich.traceback.install(**traceback_install_kwargs)  # type: ignore[arg-type]
+
+
+# @cache is supported from 3.9 onwards
```

Comment:
Nit: I would just leave out this comment; pyupgrade will take care of it when drop 3.8.